config/v1/types_image.go

```diff
@@ -34,9 +34,16 @@ type ImageSpec struct {
 	ExternalRegistryHostnames []string `json:"externalRegistryHostnames,omitempty"`
 
 	// AdditionalTrustedCA is a reference to a ConfigMap containing additional CAs that
-	// should be trusted during imagestream import.
+	// should be trusted during imagestream import, pod image pull, and imageregistry
+	// pullthrough.
 	// The namespace for this config map is openshift-config.
 	AdditionalTrustedCA ConfigMapNameReference `json:"additionalTrustedCA,omitempty"`
+
+	// Registry contains configuration that determines how the container runtime
+	// should treat individual registries when accessing images for builds+pods. (e.g.
+	// whether or not to allow insecure access).  It does not contain configuration for the
+	// internal cluster registry.
+	Registry Registry `json:"registry,omitempty"`
```

Comment:
in this context, would TrustedRegistrySources be a better name?

Reply:
Considering that one of the sub-fields is "BlockedRegistries" i'd say "Trusted" is inappropriate

I could get behind RegistrySources though.

Reply:
:+1: to RegistrySources

Reply:
yes to RegistrySources as well. I think we'll want to keep AllowedRegistriesForImport separate - this governs the imagestream import behavior and has no bearing on the runtime registry _sources_.

Reply:
i am fine w/ RegistrySources